README.md

```diff
@@ -59,8 +59,8 @@ described in detail below.
 
 1. [Sign up for IBM Bluemix](#1-sign-up-for-ibm-bluemix)
 1. [Create Bluemix services](#2-create-bluemix-services)
-1. [Configure Watson Conversation Application](#3-configure-watson-conversation-application)
-1. [View Conversation Intents, Entities and Dialog](#4-view-conversation-intents-entities-and-dialog)
+1. [Configure Watson Assistant Application](#3-configure-watson-conversation-application)
```

Comment:
the links here and below will break need to update the lower case `conversation` to `assistant`